### DISCUSSION TITLE
Should we stop trading on jQuery's name?

### DISCUSSION THREAD

**Discussion by jonnybot0** [apache/groovy-geb#193] [Ideas]
First, let me say, I still think jQuery did a _lot_ to advance scripting in the web browser. That said, it's less popular today than it was when Geb was released.

Instead of talking about "the elegance of [jQuery](http://jquery.com/) content selection" in the docs, it makes more sense in 2024 to talk about DOM Query Selectors, like [JSoup](https://jsoup.org/) and similar libraries do. Indeed, we [support XPath](https://gebish.org/manual/current/#using-webdrivers-by-class-selectors) and other modes of selection, so it might be good to plug those.

For the [jQuery integration itself](https://gebish.org/manual/current/#jquery-integration), obviously the docs should remain unchanged, but headings like [The jQuery-ish navigator API](https://gebish.org/manual/current/#the-jquery-ish-navigator-api) date the library a bit. I think I might scratch together a PR to this end to show what I mean, but figured I'd try out the discussions feature for workshopping an idea.

Thoughts, @sdelamo & @Poundex ?

**Comment by sdelamo** [comment]
No strong feelings from me to change it or keep it.
